src/components/students-health-profile/StudentsHealthFilter.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import { useState } from "react";
+import Searchbar from "@/components/students-health-profile/Searchbar";
+
+export default function StudentsHealthFilterUI() {
```

Comment:
you can remove the UI here
```suggestion
export default function StudentsHealthFilter() {
```